README.md

```diff
@@ -34,85 +49,57 @@ Two experiments are carried out:
-
-+ Tesla V100-SXM2-32GB
-+ NVIDIA Driver 460.32.03
-+ CUDA 10.2.98
-+ GCC 8.3.0
-+ CentOS 7.9.2009
-+ FLAME GPU 2 @ [`d238333`](https://github.com/FLAMEGPU/FLAMEGPU2/tree/d238333)
-
-### Fixed Density Benchmark
-
-[![Fixed Density Benchmark](sample/figures/fixed-density--volume--step-ms--model--all.png)](sample/figures/fixed-density--volume--step-ms--model--all.png)
-[![Fixed Density Benchmark Zoomed](sample/figures/fixed-density--volume--step-ms--model--zoomed.png)](sample/figures/fixed-density--volume--step-ms--model--zoomed.png)
 
-### Variable Density Benchmark
-[![variable-density volume](sample/figures/variable-density--volume--step-ms--density--3drtc.png)](sample/figures/variable-density--volume--step-ms--density--3drtc.png)
+[![Variable Communication Radius Benchmark](sample/figures/v100-470.82.01/alpha.2-v100-11.0-beltsoff/comm-radius--lineplot-spatial3D-bruteforce-rtc-only.png)](sample/figures/v100-470.82.01/alpha.2-v100-11.0-beltsoff/comm-radius--lineplot-spatial3D-bruteforce-rtc-only.png)
 
-The figure below was generated using:
+## Building and Running the Benchmark
 
-+ Titan V 12GB
-+ NVIDIA Driver 455.23.05
-+ CUDA 10.2.89
-+ GCC 7.5.0
-+ Ubuntu 16.04.7 LTS
-
-### Variable Communication Radius Benchmark
-[![Variable Communication Radius Benchmark](sample/figures/comm-radius--lineplot-spatial3D-bruteforce-rtc-only.png)](sample/figures/comm-radius--lineplot-spatial3D-bruteforce-rtc-only.png)
-
-## Dependencies 
-
-+ Model Compilation
-    + [CMake](https://cmake.org/download/) `>= 3.18`
-    + [CUDA](https://developer.nvidia.com/cuda-downloads) `>= 11.0` and a Compute Capability `>= 3.5` NVIDIA GPU.
-        + CUDA `>= 10.0` currently works, but support will be dropped in a future release.
-    + C++17 capable C++ compiler (host), compatible with the installed CUDA version
-        + [Microsoft Visual Studio 2019](https://visualstudio.microsoft.com/) (Windows)
-        + [make](https://www.gnu.org/software/make/) and [GCC](https://gcc.gnu.org/) `>= 7` (Linux)
-        + Older C++ compilers which support C++14 may currently work, but support will be dropped in a future release.
-    + [git](https://git-scm.com/)
-
-+ Plotting
-    + Python `>= 3.6`
-    + `numpy`
-    + `pandas`
-    + `matplotlib`
-    + `seaborn`
-
-## Running the Benchmark
-
-### Compilation via Cmake
-
-These instructions are for execution on Volta (`SM_70`) GPUs under linux.
+Detail of dependencies and the `cmake` build process are described in full in the [FLAMEGPU2-example-template Repo](https://github.com/FLAMEGPU/FLAMEGPU2-example-template) and are not repeated here. The benchmark should be built with seatbelts off (e.g. `-DSEATBELTS=OFF` passed to the `cmake` configuration step) to disable additional run-time checks and optionally disabling Python Swig support which is not needed for this model(-DBUILD_SWIG_PYTHON=OFF ). E.g. for Volta (`SM_70`) GPUs under Linux.
 
 ```bash
+# Configure 
```

Comment:
This either needs pairing with a matching `# Build` for the second line

Reply:
done